pootle/core/views.py

```diff
@@ -464,7 +465,8 @@ def is_admin(self):
 
     @property
     def language(self):
-        return None
+        if self.tp:
+            return self.tp.language
```

Comment:
It was returning `None` because the other descendants overwrite it when necessary, even when they don't have a TP.